tests/azure/templates/pytest_tests.yml

```diff
@@ -53,6 +53,10 @@ jobs:
     env:
       IPA_SERVER_HOST: ${{ parameters.scenario }}
       RUN_TESTS_IN_DOCKER: true
+      IPA_DISABLED_MODULES: ${{ variables.disabled_modules }}
```

Comment:
This is using `variables.disabled_modules` but not `variables.ipa_disabled_modules`.

Reply:
The correct is `ipa_*`. Fixed.